services/openshift-service-impl/src/main/resources/pipeline-template.yml

```diff
@@ -16,7 +21,12 @@ objects:
   kind: BuildConfig
   metadata:
     name: ${PROJECT}
+    annotations:
+      pipeline.alpha.openshift.io/uses: '[{"name": "jenkins", "namespace": "", "kind":
+        "DeploymentConfig"}]'
```

Comment:
@edewit where did you find the information for this specific annotation? Because I'd like to know more about them!